Framework/Algorithms/test/CreateTransmissionWorkspace2Test.h

```diff
@@ -363,6 +363,32 @@ class CreateTransmissionWorkspace2Test : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(outLam->x(0)[3], 2.0924, 0.0001);
   }
 
+  void test_two_transmission_runs_stitch_ScaleRHSWorkspace() {
+
+    CreateTransmissionWorkspace2 alg;
+    alg.setChild(true);
+    alg.initialize();
+    alg.setProperty("FirstTransmissionRun", m_multiDetectorWS);
+    alg.setProperty("SecondTransmissionRun", m_multiDetectorWS);
+    alg.setProperty("WavelengthMin", 1.5);
+    alg.setProperty("WavelengthMax", 15.0);
+    alg.setProperty("Params", "0.1");
+    alg.setPropertyValue("ProcessingInstructions", "2");
+    alg.setPropertyValue("OutputWorkspace", "outWS");
+    alg.setProperty("ScaleRHSWorkspace", false);
+    alg.execute();
+    MatrixWorkspace_sptr outLam = alg.getProperty("OutputWorkspace");
+
+    TS_ASSERT_EQUALS(outLam->getNumberHistograms(), 1);
+    TS_ASSERT_EQUALS(outLam->blocksize(), 126);
+    TS_ASSERT(outLam->x(0)[0] >= 1.5);
+    TS_ASSERT(outLam->x(0)[7] <= 15.0);
```

Comment:
I think the last bin (blocksize is 126) must be smaller than 15.0 and all other bins (including the 7th bin)  as well. The latter is guaranteed since it is a histogram.

I would as well prefer to test all y values, and x values.  Since using WorkspaceCreationHelper, that should be possible. I see that x increases by 0.1 [Angstroms].

Reply:
I agree with your suggestions. After looking at the test I have removed the property params as it is optional and tested elsewhere (and this makes the numbers nicer). I have changed the test so it checks the last bin edge instead. Since the y-values are all the same I have just tested a selection - I hope you think this is suitable!

Reply:
Only to think ahead: WavelengthMin and WavelengthMax are as well not subject to test here.

Initially I thought about using a for-loop for testing all x and y values. This would be explicit, and reveals something. It's short.